src/components/Card/Card.style.ts

```diff
@@ -0,0 +1,13 @@
+import { css, SerializedStyles } from '@emotion/core';
+import { Props } from './Card';
+import { Theme } from '../../theme';
+import { Elevation } from 'index';
+
+export const cardStyle = ({ elevated }: Props) => (theme: Theme): SerializedStyles => css`
+  display: inline-block;
```

Comment:
i am not so sure that changing native block to inline-block is a good idea. I would rather like to see the parent making the adjustments, something like the below e.g based on your card showcase

```
const CardShowcase = () =>
  elevations.map(elevation => {
    return (
      <div key={elevation} css={{ marginBottom: '40px', '& > *': { height: 100, width: 749 } }}>
        <Card elevated={elevation}>Card with {elevation} elevation</Card>
      </div>
    );
  });
```

Reply:
@panvourtsis So the logic is that the parent element will wrap around the card to give the proper dimensions? Because we defined the card as a wrapper for other elements, that's why I thought that the card's height&width will be set according to the elements in it, but I couldn't do it without `display: inline-block` because otherwise width was always at 100% of the page.

With that inner `<div/>` i tried to replicate a component with set dimensions, that the card would wrap around and hence the card would get the size of the component (aka inner div). Is that the wrong way of thinking about this?